test/controllers/tasks_controller_test.rb

```diff
@@ -109,6 +123,12 @@ def in_taxonomy_scope(organization, location = nil)
           assert_equal 2, response.body.lines.size
           assert_include response.body.lines[1], 'Child action'
         end
+
+        it 'shows duration column in exported task' do
+          task = ForemanTasks::Task.with_duration.find(FactoryBot.create(:some_task).id)
+          get(:sub_tasks, params: { id: task.id, duration: task.duration }, session: set_session_user)
```

Comment:
Sub tasks lists tasks which are children of a task specified by `id` parameter. `duration` parameter is ignored.